sdk/servicebus/service-bus/samples/typescript/tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "target": "ES2015",
     "moduleResolution": "node",
 
-    "allowSyntheticDefaultImports": true,
-
+    "esModuleInterop": true,
+    "lib": ["ESNext.AsyncIterable"],
```

Comment:
I didn't think this was necessary if you imported [core-asynciterator-polyfill](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-asynciterator-polyfill) (but I might be wrong). Do all the samples fail to compile without this?

Reply:
I think we have one iterator-based sample that would probably fail.

However, this isn't the only samples project that does it.